packages/web/components/Link/index.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import NextLink from 'next/link';
+
+const Link = ({ children, href, passHref, ...rest }) => {
+	return (
+		<NextLink href={href} passHref={passHref} {...rest}>
```

Comment:
Uma coisa interessante a se fazer é criar uma funcção `sanitizeNextLinkProps` e retornar somente as props suportadas pelo component NextLink. Dessa forma que está corremos o risco de enviar props indevidas, tipo className ou inline styles.

Reply:
Pensando bem, como são poucos parâmetros, vou desestruturar e repassar somente os parâmetros que são aceitos pelo next/link na documentação do next. É bom porque já aparece as opções no editor pra quem for utilizar o nosso link.